index.js

```diff
@@ -40,7 +43,35 @@ function toggleColor(element) {
   }
 }
 
+document.addEventListener('click', event => {
+  if (event.target === header) {
+    toggleColor(header)
+  }
+  else if (event.target.className === 'like-button') {
+    addLikes(event)
+
+  }
+})
 
 /***** Deliverable 2 *****/
 
+const newPlayerForm = (document.querySelector("#new-player-form"))
+
+newPlayerForm.addEventListener('submit', event => {
+  event.preventDefault()
+  let player = {}
+  player.number = event.target.number.value
+  player.name = event.target.name.value
+  player.nickname = event.target.nickname.value
+  player.photo = event.target.photo.value
+  player.likes = 0
```

Comment:
Looks good! You can shorten up the object initialization by writing it out like this:

```js
let player = {
    number: event.target.number.value,
    name: event.target.name.value,
    nickname: event.target.nickname.value,
    photo: event.target.photo.value,
    likes: 0
}
```